### ISSUE TITLE
Scrolling with Talkback Enabled

### ISSUE THREAD

**Issue by poloolop** [henryblue/TvRecyclerView#16]
Horizontal or Vertical scrolling with DPad doesn't work when Talkback is enabled. Tried on Nexus Player

**Comment by henryblue** [comment]
Not support mobile, you can try on android tv emulator, it's ok.

**Comment by poloolop** [comment]
I tried on Android TV Emulator also, along with Nexus Player. Scrolling stops till visible elements on the page. It doesn't scroll after last visible element on the screen, when Talkback is enabled. (Using D-Pad inputs)

**Comment by MuviDeveloper** [comment]
I have tried in Android tv and  android box , at the time of scrolling  one shadow is coming on the background of recyclerview .

**Comment by henryblue** [comment]
yes, In the automatic focus mode (the default mode), I will draw another layer on the item to mark the item is selected, so there will be overlap (if the background of the item is clear)

**Comment by MuviDeveloper** [comment]
Thank you,  for the quick response.

**Comment by MuviDeveloper** [comment]
I have one more query, after scrolling horizontally again and again it crashes,below I have mentioned the issue.

 Exception in MessageQueue callback: handleReceiveCallback
12-12 10:40:32.190 9733-9733/com.release.muvi.tv E/MessageQueue-JNI: java.lang.IllegalArgumentException: Invalid target position
                                                                         at android.support.v7.widget.RecyclerView$SmoothScroller.start(RecyclerView.java:11158)
                                                                         at android.support.v7.widget.RecyclerView$LayoutManager.startSmoothScroll(RecyclerView.java:7897)
                                                                         at app.com.tvrecyclerview.TvRecyclerView.processPendingMovement(TvRecyclerView.java:529)
                                                                         at app.com.tvrecyclerview.TvRecyclerView.isFindNextFocusView(TvRecyclerView.java:513)
                                                                         at app.com.tvrecyclerview.TvRecyclerView.processMoves(TvRecyclerView.java:613)
                                                                         at app.com.tvrecyclerview.TvRecyclerView.onKeyDown(TvRecyclerView.java:549)
                                                                         at android.view.KeyEvent.dispatch(KeyEvent.java:2694)
                                                                         at android.view.View.dispatchKeyEvent(View.java:9979)
                                                                         at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1625)
                                                                         at app.com.tvrecyclerview.TvRecyclerView.dispatchKeyEvent(TvRecyclerView.java:498)

**Comment by henryblue** [comment]
Thank you very much, you can use 1.1.8 version to try

**Comment by MuviDeveloper** [comment]
Thank you so much now it's working fine.

**Comment by poloolop** [comment]
I can confirm that scrolling works with Talkback enabled. In the new behavior, it doesn't read the text on the highlighted item, which is expected when scrolling with Talkback. Thanks for the update. I am closing the issue.
